src/core/BasicCell/style.ts

```diff
@@ -0,0 +1,111 @@
+import { styled } from "@mui/material/styles";
+import {
+  CommonThemeProps,
+  fontBodyS,
+  fontBodyXxs,
+  getColors,
+  getSpaces,
+  getTypography,
+} from "../styles";
+
+export interface BasicCellExtraProps extends CommonThemeProps {
+  textPosition?: "left" | "center" | "right";
+  shouldTextWrap?: boolean;
+  primaryTextWrapLineCount?: number;
+  secondaryTextWrapLineCount?: number;
+}
+
+const doNotForwardProps = [
+  "textPosition",
+  "primaryText",
+  "secondaryText",
+  "shouldTextWrap",
+  "shouldShowTooltipOnHover",
+  "tooltipProps",
+  "primaryTextWrapLineCount",
+  "secondaryTextWrapLineCount",
+];
+
+export const StyledTableData = styled("td", {
+  shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
+})`
+  ${(props: BasicCellExtraProps) => {
+    const { textPosition = "left" } = props;
+
+    const spacings = getSpaces(props);
+    const typography = getTypography(props);
+
+    return `
+        font-family: ${typography?.fontFamily};
+        padding: ${spacings?.l}px ${spacings?.s}px;
+        text-align: ${textPosition};
+        min-width: 96px;
+        max-width: 100%;
+        display: block;
+        overflow: hidden;
+        border: dashed 1px #ddd;
```

Comment:
do we have a predefined border for this in getBorders?

Reply:
This was for preview purposes. I forgot to remove it. Nice catch 👀